ros/src/twist_controller/dbw_node.py

```diff
@@ -54,7 +54,7 @@ def __init__(self):
         vc.vehicle_mass = rospy.get_param('~vehicle_mass', 1736.35)
         vc.fuel_capacity = rospy.get_param('~fuel_capacity', 13.5)
         vc.brake_deadband = rospy.get_param('~brake_deadband', .1)
-        vc.decel_limit = rospy.get_param('~decel_limit', -5)
+        vc.decel_limit = rospy.get_param('~decel_limit', -1.)
```

Comment:
Can you update dbw_sim.launch? The value there overrides the default specified in this line. Also, there was a snippet of John's team's config on #p-system-integration and they use accel/decel limits of 0.8 and -0.8.

Reply:
ok I'll look into it